ark/segmentation/marker_quantification_test.py

```diff
@@ -201,6 +201,27 @@ def test_create_marker_count_matrices_base():
     assert np.array_equal(normalized['chan0'], np.repeat(1, len(normalized)))
     assert np.array_equal(normalized['chan1'], np.repeat(5, len(normalized)))
 
+    # error checking
+    with pytest.raises(ValueError):
+        # attempt to pass non-xarray for segmentation_labels
+        marker_quantification.create_marker_count_matrices(segmentation_labels.values,
+                                                           channel_data)
+
+    with pytest.raises(ValueError):
+        marker_quantification.create_marker_count_matrices(segmentation_labels,
+                                                           channel_data.values)
+
+    channel_data_bad = channel_data.copy()
+    channel_data_bad = channel_data_bad.reindex({'fovs': [0, 1]})
+
+    segmentation_labels_bad = segmentation_labels.copy()
+    segmentation_labels_bad = segmentation_labels_bad.reindex({'fovs': [1, 2]})
```

Comment:
Only one of these changes should be necessary to create different FOVs, right?

Reply:
I'll remove one of these reindexes.